pjsip/include/pjsip/sip_msg.h

```diff
@@ -1165,6 +1165,18 @@ pjsip_generic_array_hdr* pjsip_generic_array_hdr_shallow_clone( pj_pool_t *pool,
 						 const pjsip_generic_array_hdr *hdr);
 
 /**
+ * Print a generic array header
+ *
+ * @param hdr	      Header to clone.
+ * @param buf	      The buffer.
+ * @param size>       The size of the buffer.
```

Comment:
Accidental ">" which has messed up your alignment.

Reply:
This is pjsip's stupid indentation system where tabs show up as '>'. I've fixed this up.